speech/tests/unit/test_helpers.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from types import GeneratorType
+import unittest
+
+import mock
+
+from google.auth.credentials import Credentials
+
+from google.cloud.speech_v1 import SpeechClient
+from google.cloud.speech_v1 import types
+
+
+class HelperTests(unittest.TestCase):
+    def setUp(self):
```

Comment:
ISTM that `setUp` is unnecessary. I (personally) bigly frown upon "stateful" test cases that don't need to be. Just use a `_get_client` helper function in this module